app/assets/javascripts/dradis/plugins/calculators/cvss/calculator.js.coffee

```diff
@@ -31,7 +40,9 @@
     # AvailabilityRequirement, ModifiedAttackVector, ModifiedAttackComplexity,
     # ModifiedPrivilegesRequired, ModifiedUserInteraction, ModifiedScope,
     # ModifiedConfidentiality, ModifiedIntegrity, ModifiedAvailability
-    output = CVSS.calculateCVSSFromMetrics(av, ac, pr, ui, s, c, i, a,
+    calc = if $('[data-behavior~=cvss-version-toggle]').prop('checked') then CVSS else CVSS31
```

Comment:
We do this check in the `init`. We should store the result and "just know" what we're initializing everywhere else so we don't have to check again.

Thinking through this, maybe it should be an initialization variable? `CVSSCalculator.init(version: CVSS31)` 

It could look a couple different ways but I'm thinking the class should be one version or the other, but never wondering what it is mid-way.

Reply:
You're right, updated.